tracee/consts.go

```diff
@@ -393,10 +393,10 @@ func init() {
 }
 
 // essentialEvents is a list of event ids (in EventsIDToName map) that are essential to the operation of tracee and therefore must be traced
-var essentialEvents = []string{
-	"do_exit",
-	"execve",
-	"execveat",
+var essentialEvents = []int32{
```

Comment:
I don't really understand the reason for moving from string to int32 - can you please explain?

Reply:
We just need to pick one type as the native type and I wasn't sure which one so I tried both. the fact that I tried first string and then int was just a coincidence. using string will not save us conversions, they will just move to other places.